### ISSUE TITLE
[iOS] Weird behavior when scrolling upon poles

### ISSUE THREAD

**Issue by Pryotic** [mousebird/WhirlyGlobe#1175]
Hello,

I have a problem where I can't find a solution for since a few hours. When I pass the poles while scrolling my globe, the globe automatically makes a weird turn. 

Here is a video showing it: (I only scroll upwards there, I don't rotate anything with gestures)

[Short Video](http://sendvid.com/so6wjq5t)

I read that similar behavior can come with the keepNorthUp property activated, but I never activated it and checking on it, I found it false as it should be.

I have a second problem which I suggest to have the same cause as the first one:
I can't rotate my Globe properly using two fingers even though rotateGesture is set to true.
I mean that kind of gesture:

[ScreenShot](https://user-images.githubusercontent.com/48635754/58721139-732ac900-83d4-11e9-8614-752fc6377a1d.jpg)

In other words: it's not possible to scroll the poles on the right or left side of the globe

I would appreciate any help a lot.

**Comment by mousebird** [comment]
You can turn off keepNorthUp (on iOS, something similar on Android) if you'd like a free rotate.

**Comment by Pryotic** [comment]
well I did deactivate it, that's the weird thing.
